nautobot/circuits/views.py

```diff
@@ -248,7 +229,7 @@ def queryset_list_url_filter(self, key, value, context):
                 section=SectionChoices.LEFT_HALF,
                 weight=100,
                 fields="__all__",
-                exclude_fields=["comments", "circuit_termination_a", "circuit_termination_z"],
+                exclude_columns=["comments", "circuit_termination_a", "circuit_termination_z"],
```

Comment:
Sorry, too far - for an ObjectFieldsPanel, keeping it as `exclude_fields` would be correct/preferable. Only ObjectsTablePanel should be changed to `exclude_columns`.